packages/cog/src/builder.ts

```diff
@@ -83,13 +87,22 @@ export class CogBuilder {
                     projection = imageProjection;
                 }
 
+                const noData = this.findNoData(tiff, logger);
+                if (noData != null && noData != nodata) {
```

Comment:
`noData` to similar to `nodata`

Reply:
changed to tiffNoData (since its the nodata value read from the tiff)